app/org/sagebionetworks/bridge/cache/CacheProvider.java

```diff
@@ -89,7 +93,42 @@ public Study getStudy(String identifier) {
     public void removeStudy(String identifier) {
         try {
             String redisKey = RedisKey.STUDY.getRedisKey(identifier);
-            stringOps.delete(redisKey).execute();
+            stringOps.delete(redisKey);
+        } catch(Throwable e) {
+            promptToStartRedisIfLocalEnv(e);
+            throw new BridgeServiceException(e);
+        }
+    }
+    
+    public String getString(String cacheKey) {
+        try {
+            String ser = stringOps.get(cacheKey);
+            if (ser != null) {
+                stringOps.expire(cacheKey, BridgeConstants.BRIDGE_SESSION_EXPIRE_IN_SECONDS);
```

Comment:
This extends the cache TTL every time a string is accessed. This is probably not desirable for the general case.

For cases where we do want to extend the cache TTL, this should be configurable, rather than hardcoded at 24 hours. (Also, the cache TTL shouldn't use the same constant as session TTL.)